build/common.sh

```diff
@@ -311,6 +311,7 @@ function kube::build::build_image() {
     api
     build
     cmd
+    contrib/mesos
```

Comment:
contrib/* should not be part of the default build.  I think it would be better to have a Makefile or something in this dir that builds everything therein.